.github/workflows/benchmark-pr.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: Benchmarks on AMD64
+permissions: read-all
+on: [pull_request]
+jobs:
+  benchmark-pull-request:
+    runs-on: ubuntu-latest-8-cores
+    steps:
+    - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+      with:
+        fetch-depth: 0
+    - id: goversion
+      run: echo "goversion=$(cat .go-version)" >> "$GITHUB_OUTPUT"
+    - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
+      with:
+        go-version: ${{ steps.goversion.outputs.goversion }}
+    - name: Run Benchmarks
+      run: |
+        BENCHSTAT_OUTPUT_FILE=result.txt make test-benchmark-compare REF=${{ github.event.pull_request.head.sha }}
```

Comment:
This should point to the base of the PR branch (`BASE_TO_COMPARE`), so it should be `github.event.pull_request.base.sha`?

Reply:
Yes, you're right. I'll update this.